docs/source/models/gbd2017/causes/diarrheal_diseases.rst

```diff
@@ -61,10 +61,75 @@ I: _I_nfected and currently experiencing a diarrheal disease bout
 
 Data Descriptions
 -----------------
+	 
```

Comment:
@NathanielBlairStahn and @kiranklc  and @yongqx have all already introduced formats for these tables, I think.  Your format is different than theirs.  See if we can figure out a way to do this consistently (check in with them at coworking maybe)

Reply:
I kind of like @beatrixkh 's format. But yes, we should standardize if possible.